src/wx/panel.cpp

```diff
@@ -1080,7 +1084,9 @@ void GameArea::OnIdle(wxIdleEvent& event)
 
     if (!paused) {
         HidePointer();
+#ifndef __WXMSW__
         event.RequestMore();
+#endif // __WXMSW__
```

Comment:
Actually use a dummy `OnIdle` event value, and this will just do nothing.

Reply:
How can I create a dummy one? I mean preferably without compiler warning messages.

Reply:
Just something like this:

```cpp
wxIdleEvent dummy;
OnIdle(dummy);
```

will work fine.